db/models.py

```diff
@@ -1 +1,58 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+    def __str__(self) -> str:
+        return f"{self.name}. {self.description}"
+
+
+class Skill(models.Model):
+    name = models.CharField(unique=True, max_length=255)
+    bonus = models.CharField(
+        "Describes what kind of bonus players can get from it",
+        max_length=255
+    )
+    race = models.ForeignKey(
+        "Race",
+        related_name="skill_set",
+        on_delete=models.DO_NOTHING
+    )
+
+    def __str__(self) -> str:
+        return f"{self.name} (bonus: {self.bonus}, race: {self.race.name})"
+
+
+class Guild(models.Model):
+    name = models.CharField(unique=True, max_length=255)
+    description = models.TextField(null=True)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Player(models.Model):
+    nickname = models.CharField(unique=True, max_length=255)
+    email = models.EmailField(max_length=255)
+    bio = models.CharField(
+        "Stores a short description provided by a user about himself/herself.",
+        max_length=255
+    )
+    race = models.ForeignKey(
+        "Race",
+        related_name="player_race",
+        on_delete=models.DO_NOTHING
```

Comment:
![B77A2895-A83B-4192-AE6F-A968A455F735](https://user-images.githubusercontent.com/88662088/229341015-0408898b-504b-432d-aac9-432f6b24c140.jpeg)
Choose another deleting type to match the task

Reply:
Can we use PROTECTED in this case? Because, if we delete the race — users can be deleted too. So, It's better if we have ability to delete the race only if it doesn't have any referenced users.